tests/utils/test_data_type_utils.py

```diff
@@ -0,0 +1,64 @@
+# SPDX-License-Identifier: Apache-2.0
+# https://spdx.dev/learn/handling-license-info/
+
+# Third Party
+import pytest
+import torch
+
+# Local
+from tuning.utils import data_type_utils
+
+dtype_dict = {
```

Comment:
**please reduce this list to 5 unique types.**
 Purpose of unit test is to test OUR code path, all the different strings go through same codepath and are internally handled by gettattr() function in python to get the torch.dtype. We are not testing other Python provided utilities to see if that works. we are just testing our wrappers - do they pass code correctly and return correct types